config-templates/ibm/ess/1.1/storage-class-template.yaml

```diff
@@ -0,0 +1,19 @@
+---
+#apiVersion: storage.k8s.io/v1
+#kind: StorageClass
+#metadata:
+#   name: ibm-spectrum-scale-csi-lt
+#provisioner: spectrumscale.csi.ibm.com
+#parameters:
+#    volBackendFs: "{{ vol-backend-fs }}"
+#    volDirBasePath: "{{ vol-dir-base-path }}"
+#reclaimPolicy: Delete
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: ibm-spectrum-scale-csi-fileset
```

Comment:
make name: ibm-spectrum-scale-csi-fileset as variable like `name: {{name}}`.

Also add variable for is_default_class. Please refer this https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/netapp/netapp-ontap-san/20.07/storage-class-template.yaml